public/app/features/alerting/unified/utils/receiver-form.ts

```diff
@@ -119,6 +122,55 @@ export function formValuesToCloudReceiver(
   return recv;
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function convertJiraFieldToJson(object: Record<string, any>, type: string) {
+  // Only for cloud alert manager. Jira fields option can be a nested object. We need to convert it to JSON.
+  if (type !== 'jira') {
+    return object;
+  }
```

Comment:
I would be inclined to move this logic outside of the method, so it isn't tied to specifically "jira" within the method? So the logic of "don't do anything if isn't Jira" is clearer when this is used